23_12_week2/wangon/11497.cpp

```diff
@@ -0,0 +1,40 @@
+#include <algorithm>
+#include <cstdlib>
+#include <iostream>
+#include <string>
+#include <vector>
+#include <deque>
+using namespace std;
+
+int main()
+{
+	int t;
+	cin >> t;
+	while (t--)
+	{
+		int n;
+		cin >> n;
+		int ret = -1;
+		vector<int> arr(n);
+		deque<int> tmp;
+		for (int i = 0; i < n; i++)
+			cin >> arr[i];
+		sort(arr.begin(), arr.end());
+		tmp.push_back(*(arr.end() - 1));
```

Comment:
```suggestion
		tmp.push_back(arr.back());
```

이런 함수가 있다는 사실 알고 계셨나용 ?

Reply:
허거덩 end만 생각했는데 눈물 주르륵 ,,, 감사합니다